app/scripts/modules/core/src/pipeline/manualExecution/Triggers.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import Select, { Option } from 'react-select';
+import { head } from 'lodash';
+import { FormikProps } from 'formik';
+
+import { IPipelineCommand, ITrigger } from 'core/domain';
+import { StandardFieldLayout } from 'core/presentation';
+
+import { ITriggerTemplateComponentProps, TriggerTemplate } from './TriggerTemplate';
+
+export interface ITriggersProps {
+  triggerChanged: (t: ITrigger) => void;
+  triggerComponent?: React.ComponentType<ITriggerTemplateComponentProps>;
+  triggers: ITrigger[];
+  formik: FormikProps<IPipelineCommand>;
+}
+
+export interface ITriggersState {
+  command: IPipelineCommand;
+}
+
+export class Triggers extends React.Component<ITriggersProps, ITriggersState> {
+  constructor(props: ITriggersProps) {
+    super(props);
+    this.state = {
+      command: props.formik.values,
```

Comment:
Hmm, I'm wondering if this state is necessary at all? Cant we just wire  `props.formik.values` down to the child components?

Reply:
👍